utils/rffe_tunnel_lib.py

```diff
@@ -0,0 +1,52 @@
+import paramiko
+from rffe_nuttx_lib import *
+
+class SSHTunnel:
+    """
+    Establishes an SSH tunnel using Paramiko for forwarding data
+    between a local machine and a remote host/port via direct-tcpip.
+    """
+    def __init__(self, host, username, key_path, remote_host, remote_port, password):
```

Comment:
Make it clear that the expected password is the encryption password of the ssh private key.